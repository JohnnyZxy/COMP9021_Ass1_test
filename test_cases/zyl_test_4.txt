Yesterday, I visited Sirs Ahri and Akali. I asked Sir Akali 
who he was, and he answered impatiently: 
"At least one of Ahri and Darius is a Knight!"
 Then I met Sir Ashe who introduced me to his wife
and told me: "Exactly one of Darius and I is a Knave."
 "I am a Knave," whispered Sir Darius. 
Should I trust them? 
