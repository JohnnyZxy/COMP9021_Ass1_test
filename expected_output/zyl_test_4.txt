Which text file do you want to use for the puzzle? The Sirs are: Ahri Akali Ashe Darius
There is no solution.